README.md

```diff
@@ -1,3 +1,32 @@
 # loopback-next-extension-starter
 
 A starter project to create extensions for loopback-next.
+
+This project demonstrates how to write your own extension by implementing a basic extension that logs a class member (function). It logs the arguments for the function, the result produced by it and the time it took to produce the result. 
+
+You can set the log level for different methods by passing it into the decorator. 
```

Comment:
I would either:
- Create a `## Basic Usage` section and give a tiny example that fits with what's said on this line.
- Create a `## Config` section, and ensure that there is a separate example for how this is used.

It depends on how much config they can perform, IMO. If it's a simple matter of demonstrating all of the available features in a single snippet, then a plain explanation of the usage with a code snippet is more than enough. If there are a ton of non-conflicting configuration options, then I'd go with a config section.